addons/web/static/src/views/sample_server.js

```diff
@@ -467,7 +467,11 @@ export class SampleServer {
         const measures = [];
         for (const measureSpec of params.fields || Object.keys(fields)) {
             const matches = measureSpec.match(MEASURE_SPEC_REGEX);
-            const { fieldName, aggregateFunction, measure } = (matches && matches.groups) || {};
+            let { fieldName, aggregateFunction, measure } = (matches && matches.groups) || {};
+            if (!aggregateFunction && measureSpec in fields && fields[measureSpec].group_operator) {
```

Comment:
use fieldName instead of measureSpec? the test measureSpec in fields seems to be done only to make sure that
some tests are not broken by this code. This is not really natural

Reply:
> use fieldName instead of measureSpec?

Yes, we can. It is equivalent in fact. 🙂 

> the test measureSpec in fields seems to be done only to make sure that some tests are not broken by this code

Indeed, so you think it is better to assume we have the field in `fields`? 🙂

Reply:
fieldName can be different from measureSpec.  measureSpec has less chance to be found in fields. fieldName is supposed to be a field name :-). So I wouldn't say that both tests are equivalent. But you are right that is is necessary to test at least fieldName in fields (`if (!fieldName && !measure) {` below)

Reply:
I mean `fieldName === measureSpec` if `Boolean(!aggregateFunction) === true`. 🙂